cfme/tests/cloud/test_delete_cloud_object.py

```diff
@@ -48,7 +47,8 @@ def test_delete_instance(setup_provider, provider):
     """
     instance_name = provider.data['remove_test']['instance']
     test_instance = VM.factory(instance_name, provider)
-    test_instance.delete()
+    delete_text = 'Remove selected item'
```

Comment:
Knowledge of the test_button should not be required inside the test, we need to figure this out in the module somehow